File: Corzbank.Data.Entities/DTOs/TransferDTO.cs
using Corzbank.Data.Enums;
using System;
using System.Collections.Generic;
using System.Text;

namespace Corzbank.Data.Models.DTOs
{
    public class TransferDTO
    {
        public TransferType TransferType { get; set; }

        public string ReceiverPhoneNumber { get; set; }

        public decimal Amount { get; set; }

        public string Note { get; set; }

        public Guid SenderCardId { get; set; }

        public string ReceiverCardNumber { get; set; }
    }
}

File: Corzbank.Web/ClientApp/src/app/data/dtos/transfer.dto.ts
import { Guid } from "guid-typescript";
import { TransferType } from "../enums/transfer-type.enum";

export class Transfer {
	id: Guid;
	transferType: TransferType;
	receiverPhoneNumber: string;
	amount: number;
	date: Date;
	note: string;
	isSuccessful: boolean;
	senderCardId: Guid;
	receiverCardId?: Guid;
}
